pkg/sdk/testint/application_packages_integration_test.go

```diff
@@ -0,0 +1,325 @@
+package testint
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+todo: add tests for:
```

Comment:
In which SNOW-xxxxxx will we follow-up with this TODO?

Reply:
i haven't created a jira for this work. although since @sfc-gh-jcieslak did some work to get the second account working, this might be something we can do now.

Reply:
As for the other similar comment: we will discuss it later. You can use grouping epic SNOW-933289 to add even short subtasks like this one (sometimes comment reference is enough as a subtask description).